mod-data-import-converter-storage-server/src/main/resources/templates/db_scripts/defaultData/default_delete_marc_authority_job_profile.sql

```diff
@@ -21,37 +21,6 @@ INSERT INTO ${myuniversity}_${mymodule}.job_profiles (id, jsonb) values
   }
 }') ON CONFLICT DO NOTHING;
 
-INSERT INTO ${myuniversity}_${mymodule}.mapping_profiles (id, jsonb) values
-  ('ff029a0a-82ff-486d-b2b1-7a4ef4cb7988', '{
```

Comment:
Could you please remove also from 
org.folio.rest.impl.MappingProfileTest#defaultMappingProfileIds
org.folio.rest.impl.DataImportProfilesImpl#MAPPING_PROFILES

Reply:
done